k-distribution/tutorial/1_k/1_lambda/lesson_8/exercises/SK-combinators/tests/lambda-6.lambda.out

```diff
@@ -1 +1,3 @@
-V ==K ( SS ( SS ( SS ( KK cond ) ( KK a ) ) ( KK b ) ) ( KK c ) )
+<k>
+  SS ( SS ( SS ( KK cond ) ( KK a ) ) ( KK b ) ) ( KK c )
+</k>
```

Comment:
So are all these tests now being `krun` with llvm backend instead of `krun --pattern '<k> V:K </k>'` with Java backend basically?

Reply:
Yes